README.md

```diff
@@ -66,9 +66,16 @@ Note: integration tests are currently disabled since the backend server is curre
 ```
 
 ### Accessibility Tests
+Make sure when you create a new endpoint to add the local url to the localpa11yci.json file and the deployed url to the .pa11yci file.
+#### Locally
+```
+    pa11y-ci --config localpa11yci.json
+```
 
-1. Start the application by running `npm start` or `npm run dev`
-2. In another terminal, run the tests with this command `pa11y-ci`
+#### On Deployed Endpoints
+```
+    npm run test-accessibility
+```
```

Comment:
this still isn't ideal as we're testing things in different ways, updating two different files instead of one etc. I'd really like to run one command and either based on a url being present or checking some env variable for ENVIRONMENT aka DEVELOPMENT vs PRODUCTION I would like to run the tests.